bangazonapi/views/product.py

```diff
@@ -107,7 +107,7 @@ def create(self, request):
             new_product.clean_fields()
             new_product.save()
         except ValidationError as ex:
-            return Response({'message': ex.args[0]}, status=status.HTTP_404_NOT_FOUND)
+            return Response({'message': ex.args[0]}, status=status.HTTP_431_REQUEST_HEADER_FIELDS_TOO_LARGE)
```

Comment:
That is not an accurate status for this response. I've already suggested an accurate one in the PR comments.